src/Themes/Navigation.ios.js

```diff
@@ -37,13 +37,18 @@ export const appStyle = {
 export const tabs = [
```

Comment:
This files fixes @taniadgv [question](https://github.com/CaronaBoard/caronaboard-native/pull/9#discussion_r125184942)